my-app/src/MapView.jsx

```diff
@@ -51,13 +52,18 @@ const MapView = () => {
     }
   };
 
-  const markersOverLap = (marker1, marker2) => {
-    const latDiff = marker1.position.lat - marker2.position.lat;
-    const lngDiff = marker1.position.lng - marker2.position.lng;
-    return (
-      latDiff * latDiff + lngDiff * lngDiff < overlapRadius * overlapRadius
-    );
-  };
+  function markersOverLap(marker1, marker2) {
+    console.log(marker1);
```

Comment:
Remove these console logs since it is not clear why you need to have them

Reply:
I removed this in making things look prettier !